Task.4/main.js

```diff
@@ -0,0 +1,67 @@
+// addNumbers = 0.1 + 0.2;
+// console.log(+addNumbers.toFixed(1));
+
+// const userLogin = prompt('Enter your Login:');
+// const userPassword = prompt('Enter your password:');
+
+// if (userLogin === 'admin' && userPassword === '12pass33210') {
+//     console.log("Login is successful");
+// } else if (userLogin !== 'admin' && userPassword !== '12pass33210') {
+//     console.log("You do not have access to admin panel");
+// } else {
+//     console.log("Problem with login or password");
+// }
+
+
+
+
+// const watermelonPrice = 21,
+//     carrotPrice = 8,
+//     potatoPrice = 12,
+//     tomatoPrice = 20;
+
+// let selectedProduct;
+// let productQuantity;
+// let userBalance;
+// let userMoney = +prompt('how much money do you have?');
+
+// if (userMoney >= watermelonPrice && userMoney >= carrotPrice && userMoney >= potatoPrice && userMoney >= tomatoPrice) {
+//     console.log(`Price: watermelon: 21 UAH, carrot: 8 UAH, potato: 12 UAH, tomato: 20 UAH.`);
```

Comment:
You already have prices declared 
```suggestion
//     console.log(`Price: watermelon: ${watermelonPrice} UAH, carrot: ${carrotPrice} UAH, potato: ${potatoPrice} UAH, tomato: ${tomatoPrice} UAH.`);
```